addons/mrp/static/src/components/bom_overview_display_filter/mrp_bom_overview_display_filter.xml

```diff
@@ -2,10 +2,10 @@
 <templates xml:space="preserve">
 
     <t t-name="mrp.BomOverviewDisplayFilter" owl="1">
-        <Dropdown class="'btn-group'" togglerClass="'btn btn-secondary'">
+        <Dropdown class="'btn-group flex-grow-1 flex-md-grow-0'" togglerClass="'btn btn-secondary'" showCaret="true">
             <t t-set-slot="toggler">
                 <span class="fa fa-filter"/>
-                Display: <span class="text-wrap" t-esc="currentDisplayedNames"/>
+                Filters
```

Comment:
Could you leave the wording `Display` instead of `Filters` ? Also the `currentDisplayedNames` getter is no more used, we can remove it.